### DISCUSSION TITLE
表头分组功能有更简洁的方式处理吗?

### DISCUSSION THREAD

**Discussion by densen2014** [dotnetcore/BootstrapBlazor#319] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/8428709/212745488-5b9fbbb8-a5e5-4aa6-b316-857eac3d6021.png)

例如下面我还是想要全部字段显示, 能不能拉姆达处理处理?

**Comment by ArgoZhang** [comment, marked as answer]
最近正在整理 Table 的示例尽量做到 CV 即可
